gnomad_mitochondria/pipeline/combine_vcfs.py

```diff
@@ -278,9 +278,9 @@ def apply_mito_artifact_filter(
             hl.locus("MT", mt.locus.position, reference_genome="GRCh37"),
             hl.locus(
                 "MT",
-                mt.locus.position + hl.len(mt.alleles[0]),
+                mt.locus.position + hl.len(mt.alleles[0]) - 1,
```

Comment:
Sorry, can you just explain why you need these two changes to interval, just so I understand the logic?

Reply:
i previously was using the default intervals, but because they are open at the end, if there's a variant at the last position of the mitochondria, it will error out but the interval will include a position not in the reference

Reply:
Ah, OK, that makes sense, thank you!